Makefile

```diff
@@ -178,12 +178,14 @@ clean:
 	rm -rf $(OUT_DIR)
 .PHONY: clean
 
-# Build a release of OpenShift for linux/amd64 and the images that depend on it.
+PLATFORM=$(shell bash -c 'source hack/common.sh; os::build::host_platform')
```

Comment:
I have the following feedback:
 - you're already calling to a shell to do the subshell (`$( )`), so it's a little overkill to call a subshell from within that, as well
 - nowhere else in our tooling do we attempt to support multiple shells, so I don't think we should try to here -- simply `source` and if it's not a Bourne Again Shell doing the `source`, oh well
 - we should allow the user to provide a `$PLATFORM` and only set this as a default
 - calling in to `hack/common.sh` seems like the wrong flow of information -- elsewhere information flows Makefile --> Bash

I would suggest supporting `OS_ONLY_BUILD_PLATFORMS='host'` or something inside the scripts to mean "build only for the output of `os::build::host_platform`.